client/src/rpc_client.rs

```diff
@@ -227,6 +227,7 @@ impl RpcClient {
                         for (i, log) in logs.iter().enumerate() {
                             debug!("{:>3}: {}", i + 1, log);
                         }
+                        debug!("");
```

Comment:
This is weird, without an extra `debug!()` the final log on line 228 is lost in the cli output.  I didn't care to debug yet why so just snuck in this magic line 🙇🏼

Reply:
Does flushing stderr have the same effect?

Reply:
It might, I didn't see a nice place to put that logic.  🤷🏼, this fixes my wagon